sites/header/deprecated/header-v1.html

```diff
@@ -0,0 +1,64 @@
+---
+{
+	"layout": "false",
```

Comment:
Have you tried to pass the `false` value instead of the string`"false"`. Because the layout named "false" don't exist in our GCWeb Jekyll instance

Reply:
As soon as the server is back for testing, I will update my branch and re-test it.